holidays/countries/svalbard_and_jan_mayen.py

```
@@ -0,0 +1,47 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)